src/server/api/lib/fakeservice.js

```diff
@@ -45,22 +46,20 @@ async function convertMessagePartsToMessage(messageParts) {
     assignment_id: lastMessage.assignment_id,
     service: "fakeservice",
     send_status: "DELIVERED"
-  });
+  };
 }
 
 async function handleIncomingMessage(message) {
   const { contact_number, user_number, service_id, text } = message;
-  const pendingMessagePart = new PendingMessagePart({
+  const [{ id: partId }] = await r.knex("pending_message_part").insert({
     service: "fakeservice",
     service_id,
     parent_id: null,
     service_message: text,
     user_number,
     contact_number
   });
```

Comment:
Does this need a `.returning('id')`, or is the default `.returning('id')` if nothing else is specified?